automated install/basic-install.sh

```diff
@@ -98,6 +98,12 @@ if command -v apt-get &> /dev/null; then
     return 0
   }
 
+  source /etc/os-release || true
+    if [[ ${PRETTY_NAME} == "Raspbian GNU/Linux 8 (jessie)" ]]; then
```

Comment:
Will this finally break support for `wheezy`?

And what about `stretch`? Will it still be using `dhcpcd5`?